scripts/wizard.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+set -e
+########################################
+#             Please Read!             #
+# This script will not run standalone, #
+# please use launch.sh.                #
+########################################
+
+#########################################################
+# helper functions                                      #
+#########################################################
+
+print_user_input_query() {
+    local question="$1"
+    local default_answer="$2"
+    local msg="\033[1;36m${question}\033[0m"
+    [[ ! -z "$default_answer" ]] && msg="${msg} [\033[2;37m${default_answer}\033[0m]"
+    msg="${msg}:"
+    echo -e "${msg}"
+}
+
+print_error() {
+    local msg="$1"
+    echo -e "\033[1;31m${msg}\033[0m"
+}
+
+#########################################################
+# end of helper functions                               #
+#########################################################
+
+# Wizard Code
+AVAILABLE_CONFIGS=("godot" "unreal")
+
+print_green "Welcome to the unifree Wizard!"
+print_green "Please answer a couple of questions so that we can get you started!"
+
+# Ask for OpenAPI Key
+if [[ -z "$OPENAI_API_KEY" ]]; then
+    while [[ $OPENAI_API_KEY_TMP = "" ]]; do
+        print_user_input_query "What is your OpenAPI key?"
+        read OPENAI_API_KEY_TMP
+    done
+    OPENAI_API_KEY="${OPENAI_API_KEY_TMP}"
+else
```

Comment:
Do we need this else case? If the env variable is defined, what will this block do?

Reply:
The idea was that the user could see the value that was set in the `[]` blocks and choose whether to change the value or keep it as is.
I can change it if it isn't ideal

Reply:
nvm, I see what you're saying :P I'll take care of it